<launch> 

	<param name="latitude_init" value="45.6216561271"/>
	<param name="longitude_init" value="9.28155230131"/>
	<param name="h0" value="224.616616895"/>
	
        <param name="minSafe" value="5.0"/>
	<param name="minUnSafe" value="1.0"/>
	<param name="minCrash" value="0.0"/>


	<node pkg="first_project" name="front_converter" type="converter" output="screen" args = "front"
	/>
	<node pkg="first_project" name="obs_converter" type="converter" output="screen" args = "obs">
	<remap from ="/swiftnav/front/gps_pose"  to="/swiftnav/obs/gps_pose"/>
	</node>

 	<node pkg="first_project" name="distance_calculator" type="distance_calculator" output="screen"
	/>

	<node pkg="first_project" name="distance_publisher" type="distance_publisher" output="screen"
	/>


</launch>
